apps/rule-manager/conf/routes

```diff
@@ -3,49 +3,54 @@
 # https://www.playframework.com/documentation/latest/ScalaRouting
 # ~~~~
 
-GET     /                           controllers.HomeController.index()
-GET     /oauthCallback              controllers.HomeController.oauthCallback()
-# A temporary endpoint to test permissions
-GET     /editPermissionCheck        controllers.HomeController.editPermissionCheck()
+GET     /                               controllers.HomeController.index(path = "")
+GET     /oauthCallback                  controllers.HomeController.oauthCallback()
 
 +anyhost
 GET     /healthcheck                controllers.HomeController.healthcheck()
 +nocsrf
-POST    /refresh                    controllers.RulesController.refresh()
+POST    /api/refresh                    controllers.RulesController.refresh()
 +nocsrf
-GET     /rules                      controllers.RulesController.list()
+GET     /api/rules                      controllers.RulesController.list()
 +nocsrf
-POST    /rules                      controllers.RulesController.create()
+POST    /api/rules                      controllers.RulesController.create()
 +nocsrf
-GET    /rules/batch/:ids           controllers.RulesController.getRules(ids: String)
+GET    /rules/batch/:ids                controllers.RulesController.getRules(ids: String)
 +nocsrf
-POST    /rules/batch                controllers.RulesController.batchUpdate()
+POST    /rules/batch                    controllers.RulesController.batchUpdate()
```

Comment:
Is there a reason these routes don't have the prefix? They came in recently so this might have slipped through the cracks

Reply:
(the corresponding fetches would need to be updated too)